.travis.yml

```diff
@@ -38,57 +38,61 @@ dist: trusty
 matrix:
   include:
     # Rails dev / 6 builds >= 2.4.4
-    - rvm: 2.6.3
+    - rvm: 2.6.6
       env: RAILS_VERSION=master
-    - rvm: 2.5.3
+    - rvm: 2.5.8
       env: RAILS_VERSION=master
-    - rvm: 2.4.4
+    - rvm: 2.4.10
       env: RAILS_VERSION=master
 
     # Rails 5.2 builds >= 2.2.2
-    - rvm: 2.6.3
+    - rvm: 2.6.6
       env: RAILS_VERSION='~> 5.2.0'
-    - rvm: 2.5.3
+    - rvm: 2.5.8
       env: RAILS_VERSION='~> 5.2.0'
-    - rvm: 2.4.4
+    - rvm: 2.4.10
       env: RAILS_VERSION='~> 5.2.0'
-    - rvm: 2.3.7
+    - rvm: 2.3.8
       env: RAILS_VERSION='~> 5.2.0'
     - rvm: 2.2.10
-      env: RAILS_VERSION='~> 5.2.0'
+      # An incompatibility with Ruby 2.2 in Rails 5.2.4.0 (still not fixed 5.2.4.3)
+      # prevents from testing against this combination. This bug is not in 5-2-stable:
+      #   lib/action_dispatch/request/session.rb:96: syntax error, unexpected '.'
+      #             id&.public_id
+      env: RAILS_VERSION='5-2-stable'
```

Comment:
On master we test both, and allow failure direct in the matrix.

Reply:
Do you suggest adding a job for `~> 5.2.0` here and allow its failure?